index.js

```diff
@@ -21,7 +21,7 @@ function reactify(filename, options) {
       try {
         var output = ReactTools.transform(source, {
           es5: options.target === 'es5',
-          sourceMap: true,
+          sourceMap: ('sourceMap' in options ? options.sourceMap : true),
```

Comment:
I think it should handle the case then options are passed through command line:
```
% browserify -t [ reactify --source-map false ] ...
```
or
```
% browserify -t [ reactify --no-source-map ] ...
```